packages/core/src/domain/session/sessionStore.ts

```diff
@@ -116,6 +115,11 @@ export function startSessionStore<TrackingType extends string>(
     if (hasSessionInCache()) {
       if (isSessionInCacheOutdated(sessionState)) {
         expireSessionInCache()
+
+        // Only renew if the session is not tracked, because we want tracked session to be renewed on user activity
+        if (sessionState[productKey] && !computeSessionState(sessionState[productKey]).isTracked) {
+          renewSessionInCache(sessionState)
+        }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏on a design perspective, it is quite bothering that we need to add a specific behaviour related to how this component his used.
Could we move this logic up?

Reply:
Those lines have been removed after discussion.